_minutes/2024-11-21-wecg.md

```diff
@@ -0,0 +1,163 @@
+# WECG Meetings 2024, Public Notes, Nov 21
+
+ * Chair: Timothy Hatcher
+ * Scribes: Rob Wu
+
+Time: 8 AM PDT = https://everytimezone.com/?t=673e7800,3c0
+Call-in details: [WebExtensions CG, 21st November 2024](https://www.w3.org/events/meetings/a97adab8-e1ae-4a2b-85cf-e6b6d3d35f00/20241121T080000/)
```

Comment:
Note: this W3 link is now 404. It used to be correct before, perhaps that is why so few people joined today?

The link will be correct for the next meetings.